CHANGES.md

```diff
@@ -10,6 +10,12 @@ $ towncrier create <PR-number>.<break|feat|fix>.md --content "Short description"
 -->
 
 <!-- towncrier release notes start -->
+## __cylc-8.2.2
+
+### Enhancements
+
+[#5733](https://github.com/cylc/cylc-flow/pull/5733) - Added Hub url to global config
+
```

Comment:
We've recently switched from adding changelog entries here manually, to using "towncrier" to generate them for us.

There's some instructions for doing this in a comment at the top of this file.

I think we can get away without a changelog entry here, because it's really a cylc-uiserver feature (which we can add to the changelog at that end)